### ISSUE TITLE
support helm --namespace

### ISSUE THREAD

**Issue by maiamcc** [windmilleng/tilt#2414]
tl/dr: `helm --namespace=foo` creates namespace foo directly, w/o including a definition in the generated yaml; since Tilt's `helm()` builtin only generates yaml, it can't respect the `namespace` flag. We should maybe support the special flag `--namespace` (insert YAML with a namespace definition into the generated yaml? call `kubectl create namespace 'foo'`?).

@rally25rs was attempting to use `helm` to spin up resources in a given namespace:
```
# Tiltfile
releaseName = 'qa'
yaml = helm(
  './chart',
  name=releaseName,
  namespace=releaseName,
  values=['./values-{}.yaml'.format(releaseName)]
)
k8s_yaml(yaml)```

However, they got the following error:
```
ERROR:
       : exit status 1
       got unexpected error during build/deploy: kubectl apply:
       stderr: Error from server (NotFound): error when creating "STDIN": namespaces "qa" not found
```

I.e. the YAML generated by Helm did NOT contain a namespace definition; usually when calling `helm ... --namespace=foo`, helm creates namespace foo (if necessary) independently of generating/applying yaml. In fact, it's considered helm best practice to NOT include YAML for a namespace in your chart (see e.g. [this comment](https://github.com/helm/helm/issues/5465#issuecomment-473942223)).

we should consider supporting the `--namespace` flag to keep the tilt `helm()` builtin as close to the user's existing helm flow as possible.

**Comment by maiamcc** [comment]
whoops, this feature is already implemented, it's just not working as expected. closing this ticket.
